include/multipass/utils.h

```diff
@@ -66,7 +71,7 @@ enum class TimeoutAction
 
 // filesystem and path helpers
 QDir base_dir(const QString& path);
-multipass::Path make_dir(const QDir& a_dir, const QString& name);
+multipass::Path make_dir(const QDir& a_dir, const QString& name = QString());
```

Comment:
Why this?

Reply:
Because there are a few places that set `name` to `""` in the cal to the function and it seems silly to not just have a default empty `QString` for those cases, one of which I needed for this PR.